packages/cli/src/utils/config.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { describe, it, expect, beforeEach, afterEach } from "vitest";
+import { getConfig, saveConfig } from "./config";
+import { defaultConfig, I18nConfig } from "@replexica/spec";
+import prettier from "prettier";
+import path from "path";
+import fs from "fs";
+
+describe("@replexica/cli", () => {
+  const prettierConfigPath = path.join(process.cwd(), ".prettierrc.json");
+  const configPath = path.join(process.cwd(), "i18n.json");
+
+  afterEach(() => {
+    if (fs.existsSync(prettierConfigPath)) {
+      fs.unlinkSync(prettierConfigPath);
+    }
+    if (fs.existsSync(configPath)) {
+      fs.unlinkSync(configPath);
+    }
+  });
```

Comment:
1. Please mock filesystem during tests. Unit tests shouldn't mess with surrounding environment.
2. Add two test cases:
   - Test if i18n.json is saved in default format when no explicit prettier config exists
   - Test if i18n.json is saved using the format from explicit prettier config when one exists